packages/strapi-v4/src/dataProvider.ts

```diff
@@ -140,53 +170,53 @@ export const DataProvider = (
     },
 
     create: async ({ resource, variables }) => {
-        const url = `${apiUrl}/${resource}`;
-
+        const url = `${apiUrl}/${resource}/${id}`;
```

Comment:
```suggestion
        const url = `${apiUrl}/${resource}`;
```
This part was accidentally merged today. Then fixed in master :)